Docs/user_guide/get_help.md

```diff
@@ -0,0 +1,31 @@
+# Get Help
+
+[Contact the Slicer community or commercial partners](about.html#contact-us) if you have any questions, bug reports, or enhancement requests.
+
+## I need help in using Slicer
+
+- I have a question about how to use Slicer for a particular task or I'm not sure things are working right:
+  - Type your queston into Google. There is a good chance that your question has been asked and anwered before and all questions ever asked about Slicer are publicly available and indexed by Google.
+  - Try your best to sort out the issue by reading documentation and checking error logs
+    - User and developer manual are available at https://slicer.readthedocs.io
+    - Look at our [portfolio of training materials](https://www.slicer.org/wiki/Documentation/Nightly/Training) and make a google search to see if the question has been discussed.
+    - Check the Error Log (in menu: View / Error log) to see if there's something misconfigured, such as a directory that is not readable or writable.
+  - I am still unclear about what to do: [ask a question on the Slicer Forum](https://discourse.slicer.org)
+
+## I want to report a problem
+
+If something is definitely not working as it should then [file a bug report in the Slicer issue tracker](https://github.com/Slicer/Slicer/issues/new/choose).
+
+It is important to provide enough specific information so that a software developer can duplicate the problem. Follow instructions in the bug report template.
+
+If the problem occurs by using the application from custom code then follow [SSCCE (Short, Self Contained, Correct, Example) approach](http://sscce.org).
+
+Don't be anonymous: real people trying hard to solve real problems are more likely to get valuable help. If you tell about yourself and your project then it is more likely to get the necessary attention.
+
+## I would like to request enhancement or new feature
+
+First search on the Slicer forum to see if someone asked for this feature already. If you find a very similar request then tell us that you are interested in it, too.
+
+If you cannot find a similar topic on the Slicer forum then write a post in the [Feature request category](https://discourse.slicer.org/c/support/feature-requests)
+
+Don't be anonymous: If you write about yourself and your project then there is a higher chance that your request will be worked on. Describe what assistance you can offer for the implementation, for example your own time, expertise, funding.
```

Comment:
This very similar to one of the above lines:
`Don't be anonymous: real people trying hard to solve real problems are more likely to get valuable help. If you tell about yourself and your project then it is more likely to get the necessary attention.`

Maybe it could be in one location instead.